mobile/lib/modules/home/ui/asset_grid/asset_grid_data_structure.dart

```diff
@@ -35,19 +37,36 @@ class RenderAssetGridElement {
   });
 }
 
+enum GroupAssetsBy {
```

Comment:
May be simpler to name these enums `day` and `month` (e.g., `GroupAssetsBy.day` and `GroupAssetsBy.month`).

Reply:
In case you need to store these enums in `Hive` or something, you can also [assign values to enums in Dart 2.17](https://stackoverflow.com/questions/69294932/how-to-assign-raw-value-to-enum-in-dart)